llvm/lib/Target/AArch64/GISel/AArch64LegalizerInfo.cpp

```diff
@@ -329,6 +335,20 @@ AArch64LegalizerInfo::AArch64LegalizerInfo(const AArch64Subtarget &ST)
     return ValTy.isPointerVector() && ValTy.getAddressSpace() == 0;
   };
 
+  if (ST.hasSVE()) {
+    for (const auto OpCode : {G_LOAD, G_STORE}) {
+      getActionDefinitionsBuilder(OpCode)
```

Comment:
IIRC, you cannot do `getActionDefinitionsBuilder` for the same opcode twice, SVE and non-SVE.

Reply:
Oh..

Reply:
Look for G_ABS.

Reply:
```C++
LegalizeRuleSet &LegalizerInfo::getActionDefinitionsBuilder(unsigned Opcode) {
  unsigned OpcodeIdx = getActionDefinitionsIdx(Opcode);
  auto &Result = RulesForOpcode[OpcodeIdx];
  assert(!Result.isAliasedByAnother() && "Modifying this opcode will modify aliases");
  return Result;
}
```
`getActionDefinitionsBuilder` just returns a reference so I think doing it twice is not a problem.

Reply:
Changed accordingly